_config.yml

```diff
@@ -84,6 +84,7 @@ lattes_id: # your ID on Lattes (Brazilian Lattes CV)
 osf_id: # your OSF ID
 research_gate_profile: # your profile on ResearchGate
 scopus_id: # your profile on Scopus
+flickr_url: # your flickr URL
```

Comment:
Instead of setting the entire flickr url here, it would be better if you set just the user, and then in `_includes/social.liquid` construct the url from the user, as it is done with most of the other socials.

Reply:
Sure.